tools/tests/cordova-hcp.js

```diff
@@ -39,3 +39,50 @@ selftest.define(
 
     run.stop();
 });
+
+selftest.define(
+  "cordova CORDOVA_COMPATIBILITY_VERSION_* works", ["cordova", "slow"], function () {
+    var s = new Sandbox();
+    var run;
+
+    var androidCompatibilityVersion = '2.0';
+
+    // Override the compatibility version for android with CORDOVA_COMPATIBILITY_VERSION_ANDROID.
+    s.env.CORDOVA_COMPATIBILITY_VERSION_ANDROID = androidCompatibilityVersion;
+
+    s.createApp("myapp", "standard-app");
+    s.cd("myapp");
+
+    var platforms = s.read(".meteor/platforms");
+    s.write(".meteor/platforms", platforms + "\nandroid\n");
+
+    run = s.run("run", "android", "--mobile-server", "example.com");
+    run.waitSecs(30);
+    run.match("Started your app");
+
+    var result = JSON.parse(httpHelpers.getUrl(
+        "http://localhost:3000/__cordova/manifest.json"));
+
+    // Check in the manifest if the overriden version was used.
+    selftest.expectEqual(result.cordovaCompatibilityVersions.android, androidCompatibilityVersion);
+
+    run.stop();
+    // Save the iOS compatibility version.
+    var iosCompatibilityVersion = result.cordovaCompatibilityVersions.ios;
+
+    // Now exclude one of the plugins from compatibility version calculation.
+    s.env.CORDOVA_COMPATIBILITY_VERSION_EXCLUDE = 'cordova-plugin-meteor-webapp';
```

Comment:
No, added. But since there is no easy way of actually checking if those were excluded the only thing you actually test is whether the syntax does not brake the functionality.